WordPress/Classes/ViewRelated/Reader/ReaderRetryFailedImageView.swift

```diff
@@ -0,0 +1,110 @@
+//
+//  ReaderRetryFailedImageView.swift
+//  WordPress
+//
+//  Created by Paul Von Schrottky on 10/5/19.
+//  Copyright © 2019 WordPress. All rights reserved.
+//
+
+import UIKit
+import Gridicons
+
+protocol ReaderRetryFailedImageDelegate: AnyObject {
+    func didTapRetry()
+}
+
+class ReaderRetryFailedImageView: UIView {
+
+    private class LinkTextView: UITextView {
+        override func selectionRects(for range: UITextRange) -> [UITextSelectionRect] {
+            return []
+        }
+
+        override func caretRect(for position: UITextPosition) -> CGRect {
+            CGRect.zero.offsetBy(dx: .greatestFiniteMagnitude, dy: .greatestFiniteMagnitude)
```

Comment:
Either both methods should omit `return` statements here, or both should use them. My feeling is that it's best here to use `return` statements because that way when a new line is added there is just a single line change in the diff.

Reply:
Resolved using explicit return statements.